src/oceans/i18n.js

```diff
@@ -0,0 +1,12 @@
+import data from '../../i18n/oceans.json';
+import MessageFormat from 'messageformat'
+
+let messages;
+export const init = (i18n = {}) => {
+  const mf = new MessageFormat('en');
+  messages = {...mf.compile(data), ...i18n};
+};
+
+export const t = (key, options) => {
+    return messages[key](options);
```

Comment:
it'd be nice to have something here like
```suggestion
    if (!messages) {
      throw "I18n must be initialized before it can render translations"
    }
    return messages[key](options);
```